src/main/java/games/strategy/triplea/ui/BattlePanel.java

```diff
@@ -219,12 +218,8 @@ protected JFrame getBattleFrame() {
 
   public void listBattle(final GUID battleId, final List<String> steps) {
     if (!SwingUtilities.isEventDispatchThread()) {
-      try {
-        // recursive call
-        SwingUtilities.invokeLater(() -> listBattle(battleId, steps));
-      } catch (final Exception e) {
-        ClientLogger.logQuietly(e);
-      }
+      // recursive call
+      SwingUtilities.invokeLater(() -> listBattle(battleId, steps));
```

Comment:
`SwingUtilities#invokeLater()` doesn't throw, so I simply removed the try-catch block.